test/JDBC/input/functions/indexproperty.sql

```diff
@@ -0,0 +1,85 @@
+-- should auto-name the primary key index as t1_pkey
+create table t1(c1 int PRIMARY KEY (c1))
+go
+
+-- explicitly create index
```

Comment:
Sorry I forgot to mention earlier, Let's add these tests in our prepare/verify/cleanup formatted tests.

Reply:
Actually I see what you mean now. You want all the tests in `indexproperty.sql` to go in the prepare/verify/cleanup formatted tests & then want me to delete `indexproperty.sql`. Will do.

Reply:
Yes, that is what I meant. Thanks

Reply:
Done.